### PR TITLE
Implement endpoint for scenario duplication

### PR DESCRIPTION
**Pull request by danielfdsilva**
Merge after https://github.com/WorldBank-Transport/Rural-Road-Accessibility/pull/119

### CODE REVIEW

**Comment by danielfdsilva** [comment]
Commit https://github.com/WorldBank-Transport/Rural-Road-Accessibility/pull/120/commits/8874692884d34dd2650da0f2bbff27b6651b410f fixes #109

**Comment by olafveerman** [comment]
Three issues detected when duplicating a scenario:

1. in some instances, the loading indicator persists after the process finishes (error or success). This is not consistent, happens around 50% of the time.
image
2. some of the scenario creations failed because I had some old databases in the `osm-p2p-dbs` folder. Should it be part of `npm run setup` to clean to `osm-p2p-dbs` folder?

**Comment by danielfdsilva** [comment]
@olafveerman Item 1 was a problem with the frontend. Fixed in https://github.com/WorldBank-Transport/rra-frontend/commit/f79fc6970567bfc58b97b4110160ccab316e097c.

Item 2 was implemented.

File: setup/setup.js [hunk 1]
```diff
@@ -1,17 +1,43 @@
...
+const addOsmP2PData = () => {
+  console.log('Adding osm-p2p dbs:', config.osmP2PDir);
+  const src = path.resolve(__dirname, '../test/utils/data-sergipe/osm-p2p-db');
+  const copy = Promise.promisify(fs.copy);
+
+  return Promise.all([
+    copy(src, path.resolve(config.osmP2PDir, `p${1100}s${1100}`)),
+    copy(src, path.resolve(config.osmP2PDir, `p${1200}s${1200}`)),
+    copy(src, path.resolve(config.osmP2PDir, `p${1200}s${1201}`)),
+    copy(src, path.resolve(config.osmP2PDir, `p${2000}s${2000}`))
```
Comment (olafveerman) [inline setup/setup.js]: No need for placeholder here

**Comment by olafveerman** [comment]
The whole part of cleaning the osm-p2p db dir works well.

I did run into some issues with the fixtures. Running `npm run setup -- --data` populates the db folder with four databases, but when I go it iD and try to load the network, the app crashes:

```
/home/olaf/projects/Rural-Road-Accessibility/node_modules/kdb-tree-store/index.js:354
      cb(new Error('INVALID STATE'))
         ^
Error: INVALID STATE
```

**Pull request merged**